src/_gettsim_tests/test_data/renten_alter.csv

```diff
@@ -1,1369 +1,2449 @@
-2018,985,985,985,1968,1,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,986,986,986,1968,2,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,987,987,987,1968,3,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,988,988,988,1968,4,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,989,989,989,1968,5,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,990,990,990,1968,6,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,991,991,991,1968,7,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,992,992,992,1968,8,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,993,993,993,1968,9,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,994,994,994,1968,10,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,995,995,995,1968,11,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),,,,,,,,,
-2018,996,996,996,1968,12,FALSE,52,0,0,0,0,0,0,0,0,0,0,0,0,0,60,67,Regelaltersgrenze,,,Table taken from a paper which c
```

Comment:
what happened to ltsray,ltsram,wosray and wosram?
where is the variable m_pflichtbeitrag coming from?

Reply:
I had store the values of ltsray,ltsram,wosray and wosram underneath the regsray and regsram and I used `m_pflichtbeitrag` to identify the respective type of pension. So that still had to be done manually, I couldn't come up with another solution within GETTSIM. Do you have another idea?

Reply:
This is too cryptic for me to comment on in case you are valuing my opinion :laughing:

Reply:
ltsray,ltsram,wosray and wosram are original column names from the seibold table. (long term insured retirement age year, long term insured retirement age month, womens retiremen age year...)
I do not understand how you identify the type of pension with the pflichtbeitrag_m. (Ususally people wouldnt pay any contributions while being in pension - the exceptions are probably not build in yet) I guess this needs some comment.

Reply:
[ret_testing_2(1).csv](https://github.com/iza-institute-of-labor-economics/gettsim/files/10447412/ret_testing_2.1.csv)
This is how the original file looks like. The retirement ages for the three different pension types are given in separate columns with the year and month of birth in the first two columns. 
Regelaltersgrenze: regsray (year) and regsram (month)
Altersgrenze für langjährig Versicherte: ltsray and ltsram
Altersrente für Frauen: wosray and wosram

I didn't know how to test all three types for each of the cohorts, since only the lowest is considered in `_ges_rente_altersgrenze_abschlagsfrei` in Gettsim. 

So I copied ltsray and ltsram & wosray and wosram, so that all retirement ages are stored as a separate testcase. 
I hope that makes it clearer :)

Reply:
Couldn't you test the variables: `ges_rente_regelaltersgrenze`, `ges_rente_frauen_altersgrenze`, `_ges_rente_langj_altersgrenze`?

I think we should definitly stick to the same data format as the original table. Let me know if you need to transform it in some way and get stuck.

Reply:
Apologies, cannot check right now, but don't we have the original variables before comparing and taking the lowest?

Reply:
Alrighty, I will have another try during the next days

Reply:
how does this link to the "pflichtbeitrag_m."

Reply:
In my approach, "pflichtbeitrag_m." ensured that the correct pension type was calculated and tested. But in the new approach, it won't be needed anylonger

Reply:
> Apologies, cannot check right now, but don't we have the original variables before comparing and taking the lowest?

yeah, we have: ``` _ges_rente_langj_altersgrenze ``` , ```ges_rente_frauen_altersgrenze```